plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -456,13 +358,23 @@ fu_synaptics_rmi_device_setup (FuDevice *device, GError **error)
 	g_autoptr(GByteArray) f01_product_id = NULL;
 	g_autoptr(GByteArray) f01_ds4 = NULL;
 
+	/* assume reset */
+	priv->in_backdoor = FALSE;
```

Comment:
This is a funny name.  Is it more like bootloader mode/flash mode?

Reply:
No, it's more than that, and I agree the name might give someone on the internet a nosebleed. It's basically kicking the hardware when it's in bootloader mode already so that it can squirt in some new firmware.

Reply:
Could you pick a better name perhaps for it based on everything else it can do while in this mode?  `flashing_mode`, `iep mode`, something like that?

Reply:
@blueue what about `iep_mode` -- I think @superm1 is right and someone will misunderstand `in_backdoor` to mean something bad.

Reply:
That is great. I was considering the name of "backdoor" is not appropriate before, and I agree to @superm1 about iep_mode. Thanks.

Reply:
Switched to `iep_mode` now.